app/scripts/views/complete_sign_up.js

```diff
@@ -89,12 +91,16 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
                       'afterCompleteSignUp', self.getAccount());
           })
           .then(function () {
+            var account = self.getAccount();
+
+            self._notifications.broadcast(self._notifications.EVENTS.SIGNED_IN, account.toJSON());
```

Comment:
Outside of the scope of this PR - `self.fxaClient.verifyCode` should be pushed into user.js/account.js, and this notification triggered there. Let's file a follow on issue about that.